lib/fuse.c

```diff
@@ -4702,6 +4705,12 @@ struct fuse *fuse_new_31(struct fuse_args *args,
 		}
 	}
 
+
+	if (f->conf.max_idle_threads == -1) {
+		/* If the number of idle threads was not set, default to 10. */
+		f->conf.max_idle_threads = 10;
+	}
+
```

Comment:
I was thinking about the comment you made the other day, that a user may not want any idle threads. Here I was hoping to keep the default 10 threads when the user does not specify max_idle_threads, but also allow the user to specify "max_idle_threads=0".